### PR TITLE
Add golang implementation

### PR DESCRIPTION
**Pull request by majorika**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,4 @@ A collection of awesome lotto lottery number generators ~~Just for fun~~
 - [Python](./src/lotto.py)
 - [Ruby](./src/lotto.rb)
 - [Scala](./src/Lotto.scala)
+- [Golang](./src/lotto.go)
```
Comment (originerd) [inline README.md]: Could you alphabetical order?

File: src/lotto.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	}
+
+	return strings.Join(buffer, ", ")
+}
+
+func main() {
+	lotto := NewLotto()
+
+	println(lotto.ToString())
+}
```
Comment (originerd) [inline src/lotto.go]: It will be better if you put newline at end of file.

**Review by originerd — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (majorika) [reply README.md]: Fixed

File: src/lotto.go (hunk 2 above)
Comment (majorika) [reply src/lotto.go]: Fixed! :)

**Review by originerd — APPROVED** [review]
LGTM 👍

**Pull request merged**